app/lib/common/widgets/html_editor/components/mention_item.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:acter/common/widgets/html_editor/models/mention_type.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+
+class MentionItem extends StatelessWidget {
+  const MentionItem({
+    super.key,
+    required this.mentionId,
+    required this.mentionType,
+    required this.displayName,
+    required this.avatarOptions,
+    required this.isSelected,
+    required this.onTap,
+  });
+
+  final String mentionId;
+  final MentionType mentionType;
+  final String displayName;
+  final AvatarOptions avatarOptions;
+  final bool isSelected;
+
+  final VoidCallback onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    final hasKeyboard =
+        MediaQuery.of(context).navigationMode == NavigationMode.directional;
+
+    return Container(
+      height: 60,
+      color: (isSelected && hasKeyboard)
+          ? Theme.of(context).colorScheme.primary
+          : null,
+      child: ListTile(
+        dense: true,
+        onTap: onTap,
+        contentPadding: const EdgeInsets.symmetric(horizontal: 12),
+        leading: ActerAvatar(options: avatarOptions),
+        title: Text(
+          displayName.isNotEmpty ? displayName : mentionId,
```

Comment:
The suggestion only accounts for ```displayName``` being null but it can never be that as it'll either one of empty or some string. But I discovered it can be nicely put in ```.map()``` extension (if its made nullable which is realistic case) for nullables and it looks much better for ```subtitle```:
```suggestion
          subtitle: displayName.map(
          (name) => Text(name, style: Theme.of(context).textTheme.labelMedium),
          orElse: () => null,
        ),
```

Reply:
but map also only works on null not on `isEmpty` so this won't work either.

but you could make an `extension` on `String` to do `isEmptyMap` and `isNotEmptyMap`, which are a bit more handy than the inline if ...